config/config.go

```diff
@@ -35,6 +35,11 @@ type Config struct {
 	Interactive bool           `json:"-" short:"i" long:"interactive" description:"Switch on interactive mode for gen-config, does nothing if gen-config is not set"`
 	Local       bool           `json:"-" short:"l" long:"local" description:"Switch on local mode"`
 	Secure      bool           `json:"-" short:"s" long:"secure" description:"Switch on security default parameters (tls + cluster security)"`
+	//additional log params
+	LogMaxSize    int  `json:"log_max_size_mb" long:"log-max-size-mb" description:"Max size of the log file, MB" env:"QDB_REST_LOG_MAX_SIZE_MB"`
+	LogMaxBackups int  `json:"log_max_backups" long:"log-max-backups" description:"Maximum numbers of log files to keep" env:"QDB_REST_LOG_MAX_BACKUPS"`
```

Comment:
I would use the name "retention" rather than "backups", e.g. `log_max_retention_count`.